### ISSUE TITLE
Not working with happy v17

### ISSUE THREAD

**Issue by niksmac** [lob/hapi-rate-limiter#30]
Error 

```
[1] "register" must be a Function
    at Object.exports.apply (/Users/somewhere/else/node_modules/hapi/lib/config.js:22:10)
    at internals.Server.register (/Users/somewhere/else/node_modules/hapi/lib/server.js:399:31)
    at init (/Users/somewhere/else/server.js:87:10)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
(node:2539) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
    at emitWarning (internal/process/promises.js:92:15)
    at emitPendingUnhandledRejections (internal/process/promises.js:109:11)
    at process._tickCallback (internal/process/next_tick.js:189:7)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
```

**Comment by Ciruxx** [comment]
Same here, with the usual error: 
"name" is required

Please @mgartner, do you plan to port to hapi v17?

**Comment by tlhunter** [comment]
@niksmac @Ciruxx fixed in v4.0.0... Only three years late!
